pkg/sysfs/syspath_test.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package sysfs
 
 import (
-	"github.com/openebs/node-disk-manager/blockdevice"
-	"github.com/stretchr/testify/assert"
 	"os"
+	"path/filepath"
 	"testing"
+
+	"github.com/openebs/node-disk-manager/blockdevice"
```

Comment:
:+1:

Reply:
Can we follow the "standard libraries - vendor - project", pattern for ordering imports.

Reply:
My bad. will reorder